ProjectReunion.sln

```diff
@@ -120,6 +120,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ManualTestApp", "test\TestA
 EndProject
 Project("{C7167F0D-BC9F-4E6E-AFE1-012C56B48DB5}") = "AppLifecycleTestPackage", "test\TestApps\AppLifecycleTestPackage\AppLifecycleTestPackage.wapproj", "{A3FBA80D-5B35-471F-9A45-DB4B29E195B9}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ProjectReunionTelemetryResource", "dev\ProjectReunionTelemetryResource\ProjectReunionTelemetryResource.vcxproj", "{0F2BDB3F-B785-423E-B09F-F395754ADD6B}"
+EndProject
```

Comment:
Revert. This project should not be part of ReunionCore. This is part of a separate solution that will be invoked in the aggregation step.